certified-connectors/1Me Corporate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,352 @@
+          "type": "string",
+          "x-ms-summary": "Id"
+        },
+        "Name": {
+          "description": "The Name of the Card Template.",
+          "type": "string",
+          "x-ms-summary": "Name"
+        },
+        "CreatedOn": {
+          "description": "The Date on which the card was created on.",
+          "type": "string",
+          "x-ms-summary": "CreatedOn"
+        },
+        "CardsCount": {
+          "description": "The number of Cards which have this Card Template.",
+          "type": "integer",
+          "x-ms-summary": "CardsCount"
+        }
+      }
+    },
+    "ApiResponse": {
+      "type": "object",
+      "properties": {
+        "Data": {
+          "description": "The  genaric data of Response",
+          "type": "string",
+          "x-ms-summary": "Data"
+        },
+        "isSuccess": {
+          "description": "the flag of responses Status.",
+          "type": "boolean",
+          "x-ms-summary": "isSuccess"
+        }
+      },
+      "Status": {
+        "description": "the Code of responses Status.",
+        "type": "integer",
+        "x-ms-summary": "Status"
+      },
+      "Message": {
+        "description": "the message of responses Status.",
+        "type": "string",
+        "x-ms-summary": "Message"
+      }
+    },
+    "AccountInvitations": {
+      "type": "array",
+      "properties": {
+        "Code": {
+          "description": "The code of invitation.",
+          "type": "string",
+          "x-ms-summary": "Code"
+        },
+        "Date": {
+          "description": "The Date on which the invitation was created on.",
+          "type": "string",
+          "x-ms-summary": "Date"
+        },
+        "ModifiedOn": {
+          "description": "The Date on which the invitation was Modified On.",
+          "type": "string",
+          "x-ms-summary": "ModifiedOn"
+        },
+        "CardTemplateID": {
+          "description": "The unique identifier for the Card Template.",
+          "type": "string",
+          "x-ms-summary": "CardTemplateID"
+        },
+        "NameOnCard": {
+          "description": "the Name which is written on the card.",
+          "type": "string",
+          "x-ms-summary": "NameOnCard"
+        },
+        "WorkEmail": {
+          "description": "the Work Email of the Employee.",
+          "type": "string",
+          "x-ms-summary": "WorkEmail"
+        },
+        "JobTitle": {
+          "description": "the Job Title of the Employee.",
+          "type": "string",
+          "x-ms-summary": "JobTitle"
+        },
+        "Extention": {
+          "description": "the Extention of the Employee.",
+          "type": "string",
+          "x-ms-summary": "Extention"
+        },
+        "Status": {
+          "description": "the Status of The Invitation.",
+          "type": "string",
+          "x-ms-summary": "Status"
+        }
+      }
+    },
+    "x-ms-connector-metadata": [
```

Comment:
@MICHAEL-MAURICE this property should be within the main root. please move them outside of the definition section.